docs/docs/03-main-concepts.md

```diff
@@ -42,7 +42,7 @@ After Snowpack builds your dependencies, any package can be imported and run dir
 <!-- This runs directly in the browser with `snowpack dev` -->
 <body>
   <script type='module'>
-    import * as React from 'react';
```

Comment:
I wanted to update this because of the “runs directly in the browser with snowpack dev” comment